dev-java/openjdk-jre-bin/openjdk-jre-bin-17.0.3_p7.ebuild

```diff
@@ -72,7 +72,7 @@ src_install() {
 	dodir "${dest}"
 	cp -pPR * "${ddest}" || die
 
-	java-vm_install-env "${FILESDIR}"/${PN}-${SLOT}.env.sh
+	java-vm_install-env "${FILESDIR}"/${PN}-11.env.sh
```

Comment:
Honestly I prefer this to use unique files, despite those being identical.

Reply:
This causes pkgcheck to emit a warning. We can drop the commit when rebasing if you really don't care about the warning.

Reply:
currently it does not emit this due to different  copyright year.
but does emit it for `openjdk-jre-bin`

I don't really care about this warning, but if we going to dedupe this file I'd prefer to remove version specifier altogether, so  it's clear that it's version-independent file installed with slot appended. just to avoid possible confusion in the future in case we need to make version specific modifications.